3.2/Dockerfile

```diff
@@ -1,28 +1,29 @@
+# MongoDB image for OpenShift.
```

Comment:
i think both, but you're right that we can probably remove the openshift reference at this point since this is intended to be a general rhscl docker image, not specific to openshift (though i'm guessing 95% of the time it's being used on openshift)